faker/database_structure_manager.py

```diff
@@ -0,0 +1,54 @@
+import sqlite3
+
+
+def clear_database(path_to_database: str):
+    conn = sqlite3.connect(path_to_database)
+
+    cur = conn.cursor()
+
+    cur.executescript(
+        """
+    DELETE FROM beds;
+    DELETE FROM bed_assignments;
+    DELETE FROM patients;
+    DELETE FROM patient_queue;
+    DELETE FROM sqlite_sequence WHERE name='beds';
+    DELETE FROM sqlite_sequence WHERE name='bed_assignments';
+    DELETE FROM sqlite_sequence WHERE name='patients';
+    DELETE FROM sqlite_sequence WHERE name='patient_queue';
+    """
+    )
+
+    conn.commit()
+
+    conn.close()
+
+
+def create_database_tables_structure(database_connection: sqlite3.Connection):
+    with open("init_database.sql", "r", encoding="utf-8") as file:
+        sql_script = file.read()
+
+    cur = database_connection.cursor()
+
+    cur.executescript(sql_script)
+
+    database_connection.commit()
```

Comment:
Be consistent. Use the same way for all functions:

File approach or heredoc approach.